central/reports/scheduler/schedule_test.go

```diff
@@ -0,0 +1,96 @@
+package scheduler
+
+import (
+	"bytes"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/stackrox/rox/pkg/branding"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/testutils/envisolator"
+	"github.com/stretchr/testify/assert"
+)
+
+var expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders = `
+	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedVulnReportEmailTemplateStackroxBrandingWithPlaceholders = `
+	StackRox has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+
+	To address these findings, please review the impacted software packages in the container images running within deployments you are responsible for and update them to a version containing the fix, if one is available.`
+
+var expectedNoVulnsFoundEmailTemplateRhacsBranding = `
+	Red Hat Advanced Cluster Security for Kubernetes has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+var expectedNoVulnsFoundEmailTemplateStackroxBranding = `
+	StackRox has found zero vulnerabilities associated with the running container images owned by your organization.`
+
+type vulnsAndDate struct {
+	WhichVulns string
+	DateStr    string
+}
+
+func generateExpectedVulnReportEmailTemplates(t *testing.T) (string, string) {
+	data := &vulnsAndDate{
+		WhichVulns: "for all vulnerabilities",
+		DateStr:    time.Now().Format("January 02, 2006"),
+	}
+
+	tmpl, err := template.New("VulnsRHACS").Parse(expectedVulnReportEmailTemplateRhacsBrandingWithPlaceholders)
+	assert.Nil(t, err)
```

Comment:
Please use `assert.NoError(t, err)` instead. It produces better message when there's an error.

Reply:
Okay